data-otservbr-global/scripts/quests/ferumbras_ascension/actions_zamulosh.lua

```diff
@@ -0,0 +1,116 @@
+local config = {
```

Comment:
this lever must be done with the new system, as well as the others in `scripts/actions/boss_levers`

Reply:
In fact, the new system performs all the necessary checks. In addition to being more secure and avoiding code repetition.